client/packages/programs/src/JsonForms/common/styleConstants.ts

```diff
@@ -1,3 +1,13 @@
+import { Theme } from '@mui/material';
+import { SxProps } from '@mui/system';
+
 export const FORM_LABEL_WIDTH = 40;
 export const FORM_LABEL_COLUMN_WIDTH = `${FORM_LABEL_WIDTH}%`;
 export const FORM_INPUT_COLUMN_WIDTH = `${100 - FORM_LABEL_WIDTH}%`;
+
+/** Default sx style for a single form row. */
+export const DefaultFormRowSx: SxProps<Theme> = {
+  margin: 0.5,
```

Comment:
For almost all of the usages you could add

```
  alignItems: 'center',
  display: 'flex',
  justifyContent: 'space-around',
  minWidth: '300px',
```

afaics it is only the `JSONForms/Text.tsx` and the `inputProps` on `DetailInputWithLabelRow` in `EncounterProgramEvent` which don't have these additional props. 

In my testing so far, this looks fine - and it means that you can remove a lot more of shared style items:

```
export const DefaultFormRowSpacing: SxProps<Theme> = {
  gap: 2,
  margin: 0.5,
  marginLeft: 0,
};

/** Default sx style for a single form row. */
export const DefaultFormRowSx: SxProps<Theme> = {
  alignItems: 'center',
  display: 'flex',
  justifyContent: 'space-around',
  minWidth: '300px',
  ...DefaultFormRowSpacing,
};
```


with the `EncounterProgramEvent` being

```
return (
    <DetailInputWithLabelRow
      label={label}
      inputProps={{
        value: event?.data ?? '',
        disabled: true,
        required: props.required,
        sx: DefaultFormRowSpacing,
        error: !!errors,
        helperText: errors,
      }}
      labelWidthPercentage={FORM_LABEL_WIDTH}
      inputAlignment={'start'}
    />
  );
```